tests/controllers_i_test/controllers_test.go

```diff
@@ -0,0 +1,63 @@
+package controllers_i_test
+
+import (
+	"boilerplate-api/api/controllers"
+	"boilerplate-api/api/repository"
+	"boilerplate-api/api/services"
```

Comment:
There's no such package `controllers, repository, services` to import.
getting err:
`could not import boilerplate-api/api/controllers (no required module provides package "boilerplate-api/api/controllers")`